docs/tempo/website/traceql/_index.md

```diff
@@ -0,0 +1,251 @@
+
+
+## Selecting spans
+
+TraceQL differentiates between two types of span data: intrinsics, which are fundamental to spans, and attributes, which are customizable key-value pairs. You can use intrinsics and attributes to build filters and select spans.
+
+In TraceQL, curly brackets `{}` always select a set of spans from the current trace. They are commonly paired with a condition to reduce the spans being passed in.
+
+This simple query will be evaluated on every span of every trace, one at a time.
+```
+{ .http.status = 200 }
+```
+
+If the trace being evaluated contains no spans with an attribute `http.status` with the value `200`, then no spans will be selected and this trace will not appear in the result set.
+
+If the trace does contain spans with an attribute `http.status` with the value `200`, then only those spans will be returned. The trace is reduced to only the set of spans that match the condition inside the `{}`. The result set will contain only this subset of spans matching the condition.
+
+### Intrinsic fields
+
+Intrinsic fields are fundamental to spans. These fields can be referenced when selecting spans. Note that custom attributes are prefixed with `.`, `span.` or `resource.` whereas intrinsics can be typed directly.
+
+
+| **Operation** | **Type** | **Definition**                        | **Example**            |
+|---------------|----------|---------------------------------------|------------------------|
+| status        | string   | status values are error, ok, or unset | { status = ok }        |
+| duration      | duration | end - start time of the span          | { duration > 100ms }   |
+| name          | string   | operation or span name                | { name = "HTTP POST" } |
+
+### Attribute fields
+
+Attribute fields are derived from the span and can be customized. Process and span attribute types are [defined by the attribute itself](https://github.com/open-telemetry/opentelemetry-proto/blob/b43e9b18b76abf3ee040164b55b9c355217151f3/opentelemetry/proto/common/v1/common.proto#L30-L38), whereas intrinsic fields have a built-in type. You can refer to dynamic attributes (also known as tags) on the span or the span's resource.
+
+Attributes in a query start with a scope (see Scoped attribute fields section below). To find traces with the `GET HTTP` method, your query could look like this:
+
+```
+{ .http.method = "GET" }
+```
+
+Or like this:
+```
+{ .http.method ="GET"}
+```
+
+#### Examples
+
+Find traces that passed through the prod namespace:
+```
+{ .namespace = "prod" }
+```
+
+Find any database connection string that goes to a Postgres or MySQL database:
+```
+{ .db.system =~ "postgresql|mysql" }
+```
+
+### Scoped attribute fields
+
+Attributes can be specifically scoped to either "span" or "resource". Specifying "span" or "resource" can result in significant performance benefits because it reduces the amount of data that Tempo has to retrieve to answer your query.
```

Comment:
Attributes can be unscoped if you are unsure if the requested attribute exists on the span or resource. It is encouraged to use scoped attributes as much as possible.

Reply:
Done.